### ISSUE TITLE
[DOT-128] Explore System for selecting extras

### ISSUE THREAD

**Issue by kharann** [dotkom/monoweb#360]
Questions to answer

* How do we want to convey an attendee's choices to the event host
* Which parts can the extra system be broken down into
* Can we avoid storing this as a separate table? Requires a ton of joins

### ❓ Is your feature request related to a problem? Please describe.

Sometimes, an event requires input from the user. Such as if they want to order pizza or sushi. This is called "extra", also providing extra information of a choice to the event host. Currently, there's no system for this.

### 📝 Describe the solution you'd like to see

When creating an event, the organizer can create some options which the user has to select when attending. This should preferably be stored as an array/json on the AttendanceEvent object, and the selected option should be on the Attendance object.

### 💡 Describe alternate solutions

Basically extras from ow4

### 📜 Additional information

<sub>From [SyncLinear.com](https://synclinear.com) | [DOT-128](https://linear.app/dotkom/issue/DOT-128/explore-system-for-selecting-extras)</sub>
